File: java/fizzbuzz-openclose/src/main/java/org/okiwi/codingdojo/FortyTwoRule.java
package org.okiwi.codingdojo;

/**
 * Created by sam on 2/13/17.
 */
public class FortyTwoRule implements IRule {
    @Override
    public String test(Integer number) {
        if (number == 42) {
            return "life";
        }
        return "";
    }
}

File: csharp/fizzbuzz/FizzBuzzEngine/FortyTwoRule.cs
using System;
namespace FizzBuzzEngine
{
    public class FortyTwoRule : IRule, IIsFinal
    {
        public FortyTwoRule()
        {
        }

        public string Convert(int number)
        {
            return number == 42 ? "The answer" : "";
        }
    }
}
